ted_sws/notice_publisher/services/notice_publisher.py

```diff
@@ -9,12 +9,13 @@
 from ted_sws.notice_publisher.adapters.sftp_publisher_abc import SFTPPublisherABC
 
 
-def publish_notice(notice: Notice, publisher: SFTPPublisherABC = SFTPPublisher(),
-                   remote_folder_path=config.SFTP_PATH) -> bool:
+def publish_notice(notice: Notice, publisher: SFTPPublisherABC = None,
+                   remote_folder_path: str = None) -> bool:
     """
         This function publishes the METS manifestation for a Notice in Cellar.
     """
-
+    publisher = publisher if publisher else SFTPPublisher()
+    remote_folder_path = remote_folder_path if remote_folder_path else config.SFTP_PATH
```

Comment:
Why removing default for parameter for this?
The same thing with more code.
I think it's clearer (if you still like this way) to write:
remote_folder_path = remote_folder_path or config.SFTP_PATH

VIVA non strict typed programming languages 😁